source/ShipyardPanel.cpp

```diff
@@ -353,14 +353,28 @@ void ShipyardPanel::Sell(bool toStorage)
 
 		message += "and " + to_string(count - (MAX_LIST - 1)) + " other ships";
```

Comment:
If the player is selling a large number of ships, only so many will be displayed. We should probably do the same with unique outfits so that the dialog doesn't get too large.